extensions/debuggee/src/org/exist/debugger/client/DebugClient.java

```diff
@@ -40,28 +41,29 @@
     private final static String COMMAND_STEP = "step";
     private final static String COMMAND_CONT = "cont";
 
-    private ConsoleReader console;
+    //TODO:check where that class would be
+    //private ConsoleReader console;
```

Comment:
As per previous comment...

Reply:
Follow up problem based on the missing import